### PR TITLE
GitHub Actions: Run html-validate on the entire repo

### PR DESCRIPTION
**Pull request by cclauss**
Hacking playground created with @bpmcneilly for running https://html-validate.org on some or all of our repo to find accessibility (a17y ;-) ) issues in our codebase.

### CODE REVIEW

File: .htmlvalidate.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "extends": ["html-validate:a17y"]
```
Comment (jamesachamp) [inline .htmlvalidate.json:2]: The folks at HTML-validate recommend also including the `html-validate:standard` preset (rationale [here](https://html-validate.org/rules/presets.html#html-validate-a17y)).  That may be overkill, though...

**Review by jamesachamp — COMMENTED** [review]
I love this, but it looks the validations will always fail unless we handle the false positive caused by web.py's `cond()` function.  It looks like it's possible to write a plugin for `html-validate` that could replace the entire function call with only the attribute (something similar is done with their Vue and Angular plugins).

Aside from that, I'm wondering if it's worthwhile to run these actions if no changes were made involving `.js` or `.html` files.  It looks like there is a [paths](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpaths) configuration that can be used to achieve this.

File: .htmlvalidate.json (hunk 1 above)
Comment (cclauss) [reply .htmlvalidate.json:2]: We wanted to reduce the noise (188 errors vs. 17 errors on this one directory) for our initial experimentation.

**Comment by cclauss** [comment]
@bpmcneilly Can you please give me an update on how you want to progress on this?  We could incorporate this test into our GItHub Actions with `|| true` at the end of the command.  This would allow the command to fail yet have the build pass.  This would allow you and others to see where our accessibility issues are while they are being fixed.  Once all accessibility issues were fixed, we could remove the `|| true` and require that all future code passes html-validate accessibility tests.

**Comment by cclauss** [comment]
Closing.  Can be recreated if it is needed in the future.

**Pull request closed without merging**